pkg/ddc/base/portallocator/bitmap_allocator.go

```diff
@@ -36,7 +36,7 @@ func (b *BitMapAllocator) needResetReservedPorts() bool {
 }
 
 func newBitMapAllocator(pr *net.PortRange, log logr.Logger) (BatchAllocatorInterface, error) {
-	alloc, err := portallocator.New(*pr, func(max int, rangeSpec string) (allocator.Interface, error) {
+	alloc, err := portallocator.New(*pr, func(max int, rangeSpec string, offset int) (allocator.Interface, error) {
```

Comment:
Please add TODO for passing `offset` value. Thanks.

Reply:
fixed